### PR TITLE
Added local script support for dependencies

### PR DESCRIPTION
**Pull request by ipepe**
I added support for local scripts. Also with tests. What do You think?

### CODE REVIEW

File: lib/schmooze/processor_generator.rb [hunk 1]
```diff
@@ -44,7 +44,12 @@ def generate_method(name, code)
       end
 
       def generate_import(import)
-        package, mid, path = import[:package].partition('.')
+        if import[:package][0] == '.' # if it local script else package
```
Comment (bouk) [inline lib/schmooze/processor_generator.rb]: `import[:package].start_with?('.')`

File: test/fixtures/local_script/localapp.js [hunk 2]
```diff
@@ -0,0 +1 @@
+module.exports = { test: function(){ return 456; } }
```
Comment (bouk) [inline test/fixtures/local_script/localapp.js]: missing newline

File: test/garbage_test.rb [hunk 3]
```diff
@@ -9,7 +9,7 @@ def test_process_is_not_started_until_used
     garbage = GarbageSchmoozer.new(__dir__)
     assert_nil garbage.pid
     garbage.test
-    assert garbage.pid
+    refute_nil garbage.pid
```
Comment (bouk) [inline test/garbage_test.rb]: Please remove this unrelated change

**Comment by ipepe** [comment]
@bouk Done

File: lib/schmooze/processor_generator.rb (hunk 1 above)
Comment (ipepe) [reply lib/schmooze/processor_generator.rb]: I did it like this the first time, but for some reason that I can't now remeber, I changed it to [0]. Dunno.

File: lib/schmooze/version.rb [hunk 4]
```diff
@@ -1,3 +1,3 @@
 module Schmooze
-  VERSION = "0.1.6"
+  VERSION = "0.1.7"
```
Comment (bouk) [inline lib/schmooze/version.rb]: Sorry one more change, the VERSION should be `0.2.0` because this adds a new feature

Comment (ipepe) [reply lib/schmooze/version.rb]: No problem, You are right.

**Comment by ipepe** [comment]
@bouk When it will be merged?

**Comment by bouk** [comment]
I approved it, so you can merge it

**Comment by ipepe** [comment]
@bouk Im pretty sure I can't merge it myself. I don't have permission to do that.

**Comment by bouk** [comment]
hmm it was my understanding that approving makes it so you can merge... I'll just do it!

**Comment by bouk** [comment]
This change should now be available as v0.2.0
